insets/src/main/java/dev/chrisbanes/accompanist/insets/Insets.kt

```diff
@@ -167,5 +255,12 @@ private fun Insets.updateFrom(windowInsets: WindowInsetsCompat, type: Int) {
     isVisible = windowInsets.isVisible(type)
 }
 
+internal fun Insets.coerceAtLeastEachDimension(other: Insets): Insets = copy(
+    left = left.coerceAtLeast(other.left),
+    top = top.coerceAtLeast(other.top),
+    right = right.coerceAtLeast(other.top),
```

Comment:
copypasta error here

Reply:
`top` is `right`, `right` is `top` :thinking: